include/espeak.md

```diff
@@ -98,6 +98,8 @@ It is also different from the build approach within NVDA.
 
 1. Feed the SSML into `espeak-ng.exe`
 
+   <!-- markdownlint-disable commands-show-output -->
    ```sh
    $ cat /c/work/test-espeak-ssml.txt | ./espeak-ng.exe --stdin -m
    ```
+   <!-- markdown-lint-restore -->
```

Comment:
Why not remove the dollar sign? I think the linting rule [makes sense](https://github.com/updownpress/markdown-lint/blob/master/rules/014-commands-show-output.md), alternatively, lets just disable the rule altogether

~~~suggestion
   ```sh
   cat /c/work/test-espeak-ssml.txt | ./espeak-ng.exe --stdin -m
   ```
~~~